administrator/components/com_users/src/Table/TfaTable.php

```diff
@@ -0,0 +1,433 @@
+<?php
+/**
+ * @package    Joomla.Administrator
+ * @subpackage com_users
+ *
+ * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Users\Administrator\Table;
+
+use Exception;
+use Joomla\CMS\Date\Date;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\MVC\Factory\MVCFactoryInterface;
+use Joomla\CMS\Table\Table;
+use Joomla\CMS\User\UserFactoryInterface;
+use Joomla\Component\Users\Administrator\Helper\Tfa as TfaHelper;
+use Joomla\Component\Users\Administrator\Model\BackupcodesModel;
+use Joomla\Component\Users\Administrator\Service\Encrypt;
+use Joomla\Database\DatabaseDriver;
+use Joomla\Database\ParameterType;
+use Joomla\Event\DispatcherInterface;
+use RuntimeException;
+use Throwable;
+
+/**
+ * Table for the Two-Step Verification second factor records
+ *
+ * @property int    $id          Record ID.
+ * @property int    $user_id     User ID
+ * @property string $title       Record title.
+ * @property string $method      TFA Method (corresponds to one of the plugins).
+ * @property int    $default     Is this the default Method?
+ * @property array  $options     Configuration options for the TFA Method.
+ * @property string $created_on  Date and time the record was created.
+ * @property string $last_used   Date and time the record was las used successfully.
```

Comment:
```suggestion
 * @property string $last_used   Date and time the record was last used successfully.
```
This just corrected the typo, but maybe @brianteeman has also suggestions for language.